### PR TITLE
Issue#178

### PR DESCRIPTION
**Pull request by tinchods93**
## Related Issue #178  

Closes: #178  

### Describe the changes you've made
At first, i thought of adding another element to the note list, like we have done with the title and the text content. 
But then i figured that it would be easier to just use a Class.
I also, added a function to show and hide the favorites, change the state of favorite and, by last, i added a button to the right side to filter and only show the favorites.

### Webpage
You can test the changes here:
https://tinchods93.github.io/Awesome-JavaScript-Projects/public/note-app.html

### CODE REVIEW

File: public/note-app.html [hunk 1]
```diff
@@ -54,7 +54,11 @@ <h5 class="card-title">Add a Note</h5>
         <hr>
         <h1>My Notes</h1>
         <hr>
-
+        <div class="row">
```
Comment (Vishal-raj-1) [inline public/note-app.html:57]: This button should be placed in front of My Notes above the line.

button

**Comment by tinchods93** [comment]
@Vishal-raj-1 
Ok, i already made the change, check it out:
https://tinchods93.github.io/Awesome-JavaScript-Projects/public/note-app.html

**Comment by tinchods93** [comment]
@Vishal-raj-1 Hello! Check the updates! I already made the change you metioned.

**Comment by Vishal-raj-1** [comment]
@tinchods93 Wonderful

**Pull request merged**